svthalia/concrexit
# Registration without membership

Closes #2774 



### Summary
I have added a toggle to events to enable users without memberships to register for them.

### How to test
Steps to test the changes you made:
1. Create an ex-member (or non-member) user
2. Create an event with the toggle checked
3. Observe that the ex/non-member user can register for this event, but not for any others.
4. Check that this works correctly even if data minimization has been applied to the user (@pingiun)
